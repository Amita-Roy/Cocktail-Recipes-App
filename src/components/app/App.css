.App {
  width: 1400px;
  border: 1px solid red;
  margin: 0 auto;
}
