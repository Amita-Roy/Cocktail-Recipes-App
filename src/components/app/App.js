import React from 'react';
import { Switch, Route } from 'react-router-dom';
import CocktailListPage from '../../pages/cocktailListPage/CocktailListPage';
import CocktailPage from '../../pages/cocktailPage/CocktailPage'; // eslint-disable-line import/no-named-as-default

import './App.css';

const App = () => (
  <div className="App">
    <Switch>
      <Route path="/" exact component={CocktailListPage} />
      <Route path="/cocktails/:id" exact component={CocktailPage} />
    </Switch>
  </div>
);

export default App;
