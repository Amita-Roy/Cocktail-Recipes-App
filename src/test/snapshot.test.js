import TestRenderer from 'react-test-renderer';
import { createBrowserHistory } from 'history';
import { Provider } from 'react-redux';
import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import { Router } from 'react-router-dom';
import App from '../components/app/App';
import rootReducer from '../reducers';

const history = createBrowserHistory();

const initialState = {
  cocktails: [],
  cocktail: {},
};

const composeEnhancer = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
const store = createStore(rootReducer, initialState, composeEnhancer(applyMiddleware(thunk)));

it('renders App', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <Router history={history}>
        <App />
      </Router>
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
