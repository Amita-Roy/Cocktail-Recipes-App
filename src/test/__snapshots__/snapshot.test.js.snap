// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div
  className="App"
>
  <div>
    <div
      className="jumbotron"
    >
      <div
        className="text-content"
      >
        <div
          className="title"
        >
          <h1>
            The
            <span>
              Mixing
            </span>
            <span>
              House
            </span>
          </h1>
        </div>
      </div>
    </div>
    <div
      className="cocktailsFilterForm"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          name="query"
          onChange={[Function]}
          placeholder="Filter by name"
          required={true}
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
    <div
      className="main-content"
    >
      <div
        className="cocktailList"
      >
        <div
          className="spinner-wrapper"
        >
          <div
            className="dizzy-gillespie"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
